sql_generators/urlbar_events/templates/desktop_query.sql

```diff
@@ -69,10 +69,13 @@ WITH events_unnested AS (
     COALESCE(metrics.boolean.urlbar_pref_suggest_nonsponsored, FALSE) AS pref_fx_suggestions,
     mozfun.map.get_key(extra, "engagement_type") AS engagement_type,
     mozfun.map.get_key(extra, "interaction") AS interaction,
-    CAST(mozfun.map.get_key(extra, "n_chars") AS int) AS num_chars_typed,
-    CAST(mozfun.map.get_key(extra, "n_results") AS int) AS num_total_results,
+    SAFE_CAST(mozfun.map.get_key(extra, "n_chars") AS int) AS num_chars_typed,
+    SAFE_CAST(mozfun.map.get_key(extra, "n_results") AS int) AS num_total_results,
```

Comment:
These will now be null if they have invalid integer values.  Is that OK for how these are used downstream, or should these perhaps be coalesced with zero?

Reply:
@teonbrooks do you have any preference here?

Reply:
Or alternatively, rows with invalid/null values could be filtered out of the final results if that's more appropriate.

Reply:
sorry, I had set the PR to automerge. to close the loop, I think null is appropriate here bc zero characters does have a meaning in the urlbar, it could be clicking on one of the topsites there